services/ui-src/src/components/forms/AdminBannerForm.tsx

```diff
@@ -44,7 +47,11 @@ export const AdminBannerForm = ({ writeAdminBanner, ...props }: Props) => {
       startDate: formData.startDate,
       endDate: formData.endDate,
     };
-    writeAdminBanner(newBannerData);
+    try {
+      await writeAdminBanner(newBannerData);
+    } catch (error) {
+      errorHandler(error, setErrorState, REPLACE_BANNER_FAILED);
+    }
```

Comment:
Try/catch for new error handling. Has to be async because it has to wait for `writeAdminBanner` to succeed or fail.